extensions/2.0/Khronos/KHR_materials_anisotropy/schema/glTF.KHR_materials_anisotropy.schema.json

```diff
@@ -7,11 +7,11 @@
     "properties": {
         "anisotropyStrength": {
             "type": "number",
-            "description": "The anisotropy strength. When anisotropyTexture is present, this value is multiplied by the blue channel.",
+            "description": "Anisotropic strength",
```

Comment:
The short description is usually the first statement of the detailed description. Please consider using
> The anisotropy strength.

for consistency.

Reply:
Done.